src/main/java/uk/gov/hmcts/reform/civil/validation/UnavailableDateValidator.java

```diff
@@ -50,39 +52,30 @@ public boolean isValid(UnavailableDate value, ConstraintValidatorContext context
         return errors;
     }
 
+    public List<String> validateFastClaimHearing(HearingLRspec hearingLRspec) {
+        List<String> errors = new ArrayList<>();
+        if (isFastClaimHearingNullOrEmpty(hearingLRspec) && hearingLRspec.getUnavailableDatesRequired() == YES) {
```

Comment:
I checked the method isFastClaimHearingNullOrEmpty and it doesn't check hearingLRSpec for null or not. It just check the null for unavailableDates in hearing object.
I think we should make this and similar other condition like this

`if (hearingLRspec != null && isFastClaimHearingNullOrEmpty(hearingLRspec) && hearingLRspec.getUnavailableDatesRequired() == YES)`